pkg/api/admin_users.go

```diff
@@ -30,6 +31,11 @@ func AdminCreateUser(c *models.ReqContext, form dtos.AdminCreateUserForm) {
 	}
 
 	if err := bus.Dispatch(&cmd); err != nil {
+		if err == models.ErrOrgNotFound {
+			c.JsonApiErr(400, models.ErrOrgNotFound.Error(), nil)
+			return
+		}
+
```

Comment:
I really don't agree with that. This is part of what bugged me when I was dealing with the grafana API, http codes are generally not used properly. A 500 should only be used if there was an unexpected failure on the server side. This is clearly a case for a 400 because the user send a bad parameter.

Reply:
I agree with @Sytten 
Sometimes it can be hard to choose the correct http status code but we should strive towards using them correctly.

Reply:
I'm ok with it but I would like to document it also in the API.

Reply:
To me it seems to make sense to flag that the user made a mistake in the request.